client/scala/armada-scala-client/src/test/scala/io/armadaproject/armada/ArmadaClientSuite.scala

```diff
@@ -0,0 +1,436 @@
+package io.armadaproject.armada
```

Comment:
- moved here from client/scala/armada-scala-client/src/test/scala/ArmadaClientSuite.scala
- applying scala code style
- only functional change is noted below